go-controller/pkg/ovn/service.go

```diff
@@ -105,17 +101,15 @@ func (ovn *Controller) syncServices(services []interface{}) {
 	// 'nodeportServices'.
 	gateways, stderr, err := ovn.getOvnGateways()
 	if err != nil {
-		klog.Errorf("failed to get ovn gateways. Not syncing nodeport"+
-			"stdout: %q, stderr: %q (%v)", gateways, stderr, err)
+		klog.Errorf("failed to get ovn gateways. Not syncing nodeport stdout: %q, stderr: %q (%v)", gateways, stderr, err)
```

Comment:
Ohh man, but I really get annoyed with lines that span multiple lines and makes it look like we're writing C. The problem is that go doesn't have an official opinion on how long lines should be, so does ovn-kubernetes?

Reply:
I mean we live in an era with HD :desktop_computer: with widths of 1980 px and above, we should have no problem reading long lines

Reply:
Yeah there is no line length for golang and in an IDE these days its not really a problem, but if you are reviewing code in github without going and looking at the file and looking at just the files changed tab, longer than 120 means you have to start using the slider to review code. I personally don't like that. I'm open to what other suggest, but I think there definitely needs to be some kind of line length limit we use otherwise things become unreadable. Think about how unreadable something like this would be all on one line:

https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/hybrid-overlay/pkg/controller/node_linux.go#L631

Reply:
OK, will modify this line